plugins/woocommerce/src/Admin/Features/ProductBlockEditor/ProductTemplates/GroupInterface.php

```diff
@@ -3,18 +3,17 @@
 namespace Automattic\WooCommerce\Admin\Features\ProductBlockEditor\ProductTemplates;
 
 use Automattic\WooCommerce\Admin\BlockTemplates\BlockInterface;
-use Automattic\WooCommerce\Admin\BlockTemplates\ContainerInterface;
 
 /**
  * Interface for block containers.
  */
-interface GroupInterface extends ContainerInterface {
+interface GroupInterface extends BlockInterface {
```

Comment:
You still need the ContainerInterface... looks like we have a `BlockContainerInterface` that would be exactly what we want to extend a single interface. (Note: There isn't currently a technical need, since `BlockInterface` is actually a superset of `ContainerInterface`, but if we make changes in the future we would want to make sure that `GroupInterface` is explicitly a `ContainerInterface`, via `BlockContainerInterface`).